hadoop-tools/hadoop-aws/src/test/java/org/apache/hadoop/fs/s3a/ITestS3AEndpointRegion.java

```diff
@@ -257,6 +283,83 @@ public void testWithVPCE() throws Throwable {
     expectInterceptorException(client);
   }
 
+  @Test
+  public void testCentralEndpointWithUSWest2Region() throws Throwable {
+    describe("Access bucket using central endpoint and us-west-2 region");
+    final Configuration conf = getConfiguration();
+    removeBaseAndBucketOverrides(conf, ENDPOINT, AWS_REGION);
+
+    final Configuration newConf = new Configuration(conf);
+
+    newConf.set(ENDPOINT, CENTRAL_ENDPOINT);
+    newConf.set(AWS_REGION, US_WEST_2);
+
+    newFS = new S3AFileSystem();
+    newFS.initialize(getFileSystem().getUri(), newConf);
+
+    assertOpsUsingNewFs();
+  }
+
+  @Test
+  public void testCentralEndpointWithEUWest2Region() throws Throwable {
```

Comment:
the value in having both `testCentralEndpointWithUSWest2Region` and `testCentralEndpointWithEUWest2Region` only comes in when your test bucket is in us-west-2, and then with the eu-west-2 test you check if cross region access behaves as expected. since we can't guarantee where users test against, I don't think we need both tests. Unless we end up using one of the public buckets, as for that we know the region.

Reply:
let's go with a public bucket

Reply:
public bucket wouldn't work for full CRUD operation? maybe only fs#exists and fs#open followed by some reads would work.

Reply:
yeah that's enough to know if the request is ending up in the right place. is this logic doesn't work properly, even a HEAD will fail. all we really need is a HEAD object I think

Reply:
Updated tests: accessing public bucket with central endpoint + same region, and different region, for guaranteed outcome.